website/content/docs/ecs/enterprise.mdx

```diff
@@ -121,3 +121,29 @@ module "my_task" {
 ```
 
 </CodeBlockConfig>
+
+### Audit Logging
+Consul on ECS supports [audit logging](/docs/enterprise/audit-logging) when Consul Enterprise clients are used.
+This feature has the following requirements:
+
+- ACLs must be enabled.
+- `mesh-task` must use a Consul Enterprise client image.
+- `gateway-task` must use a Consul Enterprise client image.
```

Comment:
nit: because there is one image for both clients/servers

```suggestion
- `mesh-task` must use a Consul Enterprise image.
- `gateway-task` must use a Consul Enterprise image.
```

Reply:
Suggestion added [here](https://github.com/hashicorp/consul/pull/13729/commits/a25c0c5c9aa345958fd3a0a1bcb0f1d34a00b782)